Makefile

```diff
@@ -6,8 +6,8 @@ IMAGE_BUILDER ?= podman
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
 SHAVERSION=$(shell git rev-parse HEAD)
-VERSION="latest"
-RELEASE_TAG ?= "1.0.0"
+VERSION="1.0.1"
```

Comment:
The only change that should happen here is to set:

VERSION ?= "latest"

RELEASE_TAG should not be touched. Pass those vars to make when called.

Reply:
I reverted `RELEASE_TAG` back to `0.0.0` this way local development can still happen and would be a `0.0.0` version. 
IE
```
IMAGE_REPO=acornett make build docker-build docker-push bundle bundle-build bundle-push catalog-build catalog-push
```

Then like you said a `make bundle` for a release would be `VERSION=x.y.z RELEASE_VERSION=x.y.z make bundle`